build/assets/ios/Showcase/Application/Core/Foundations/Assets/AssetsModel+Sections.swift

```diff
@@ -0,0 +1,356 @@
+            VitaminAssets.Payment.applePay,
+            VitaminAssets.Payment.bvr,
+            VitaminAssets.Payment.bancontactPayconiq,
+            VitaminAssets.Payment.bancontact,
+            VitaminAssets.Payment.belfius,
+            VitaminAssets.Payment.bitcoin,
+            VitaminAssets.Payment.bitpay,
+            VitaminAssets.Payment.cb,
+            VitaminAssets.Payment.cadhoc,
+            VitaminAssets.Payment.cash,
+            VitaminAssets.Payment.clickandBuy,
+            VitaminAssets.Payment.dinersClubInternational,
+            VitaminAssets.Payment.discover,
+            VitaminAssets.Payment.dwolla,
+            VitaminAssets.Payment.eps,
+            VitaminAssets.Payment.elo,
+            VitaminAssets.Payment.eway,
+            VitaminAssets.Payment.giroPay,
+            VitaminAssets.Payment.googlePay,
+            VitaminAssets.Payment.ing,
+            VitaminAssets.Payment.illicado,
+            VitaminAssets.Payment.ingenico,
+            VitaminAssets.Payment.jcb,
+            VitaminAssets.Payment.kbc,
+            VitaminAssets.Payment.klarna,
+            VitaminAssets.Payment.mbMultibanco,
+            VitaminAssets.Payment.mbWay,
+            VitaminAssets.Payment.mnp,
+            VitaminAssets.Payment.maestroFull,
+            VitaminAssets.Payment.maestro,
+            VitaminAssets.Payment.masterCardIDCheck,
+            VitaminAssets.Payment.mastercardFull,
+            VitaminAssets.Payment.mastercard,
+            VitaminAssets.Payment.neteller,
+            VitaminAssets.Payment.oneyClassic,
+            VitaminAssets.Payment.oney3x,
+            VitaminAssets.Payment.oney3x4x,
+            VitaminAssets.Payment.oney4x,
+            VitaminAssets.Payment.payU,
+            VitaminAssets.Payment.paylib,
+            VitaminAssets.Payment.paymill,
+            VitaminAssets.Payment.payoneer,
+            VitaminAssets.Payment.paypal,
+            VitaminAssets.Payment.paysafeCard,
+            VitaminAssets.Payment.pointsPay,
+            VitaminAssets.Payment.postFinanceCard,
+            VitaminAssets.Payment.postFinanceEFinance,
+            VitaminAssets.Payment.postePay,
+            VitaminAssets.Payment.powerPay,
+            VitaminAssets.Payment.przelewy24,
+            VitaminAssets.Payment.sepa,
+            VitaminAssets.Payment.skrill,
+            VitaminAssets.Payment.spiritOfCadeau,
+            VitaminAssets.Payment.stripe,
+            VitaminAssets.Payment.unionPay,
+            VitaminAssets.Payment.verifone,
+            VitaminAssets.Payment.visaClassic,
+            VitaminAssets.Payment.visaElectron,
+            VitaminAssets.Payment.webMoney,
+            VitaminAssets.Payment.westernUnion,
+            VitaminAssets.Payment.iDeal,
+            VitaminAssets.Payment.wallet,
+            VitaminAssets.Payment.giftCardPayment,
+            VitaminAssets.Payment.chequePayment
+        ]),
+        Assets.Section(name: "Placeholders", items: [
```

Comment:
I thinks there is an issue here it should be `AssetsModel` instead of `Assets`.

Reply:
You're right, done here: https://github.com/Decathlon/vitamin-design/pull/232/commits/bad5ac9b2a218a7fe05e5d8677656a14eb2f4d7b